modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.tsx

```diff
@@ -26,10 +26,12 @@ import type {Edge, Elements, Node} from 'react-flow-renderer';
 import type {ObjectRelationshipEdgeData} from '../types';
 
 interface RightSidebarObjectRelationshipDetailsProps {
+	learnResources: ILearnResourceContext;
 	objectRelationshipDeletionTypes: LabelValueObject[];
 }
 
 export function RightSidebarObjectRelationshipDetails({
+	learnResources,
```

Comment:
Hi @DavysonMelo, I just realized that `learnResourceContext` is available in `useObjectFolderContext`, so we don't need to receive it as a prop, we can simply destructure it from `useObjectFolderContext`